File: BirthdayCakeCandles/birthCakeCandles.py
# Level : Easy
def birthCakeCandles(candles,numberOfCandles):
    max = candles[0]
    repeatsOfMax = 0
    for candle in candles:
        if(candle > max):
            max = candle
            repeatsOfMax = 1
        elif(candle == max):
            repeatsOfMax += 1
    return repeatsOfMax        

# Examples
candles_1 = [10,2,10,2,10,4,1]
print(birthCakeCandles(candles_1,len(candles_1)))
candles_2 = [19]
print(birthCakeCandles(candles_2,len(candles_2)))

File: BirthdayCakeCandles/birthCakeCandles.rb
# Level :  Easy
# Author : RaulSosa
# Status : Solved
def birthCakeCandles(candles,numberOfCandles)
    max = candles[0]
    repeatsOfMax = 1
    candles.each do |candle|
        if(candle > max) then
            max = candle
            repeatsOfMax = 1
        elsif(candle == max) then
            repeatsOfMax += 1
        end
    end
    return repeatsOfMax
end
candles_1 = [10,10,2,2,24,4,12,24,24]

puts birthCakeCandles(candles_1,candles_1.length())
